modules/client/test/integration/multisig-client/methods.test.ts

```diff
@@ -140,7 +131,7 @@ describe("Client Multisig", () => {
       },
     };
     const ipfsUri = await multisigClient.methods.pinMetadata(metadata);
-    const endDate = new Date(Date.now() + 1000 * 60);
+    const endDate = new Date(Date.now() + 1000 * 60 * 60);
```

Comment:
You changed endDAte to new Date(0) on the other places, is there a reason why cant do it heere too?

Reply:
the multisig doesn't support setting the endDate to 0 and choose the end date automatically. So I had to increase it to make sure that it works